lang/en.json

```
@@ -158,6 +156,7 @@
         "StreamIdFilterHelpText": "The Stream ID for which you want to import data. %1$sLearn more.%2$s %3$sNote:%4$s Leave this field blank to import data from all streams.",
         "RecommendationTitle": "Import Google Analytics data to Matomo (recommended for you)",
         "RecommendationText": "We have detected Google Analytics on your site, so you can set up Matomo by importing your Google Analytics data.",
-        "RecommendationButton": "Import data"
+        "RecommendationButton": "Import data",
+        "NoteImportNotCountedAsHit": "%1$sNote:%2$s Data imported through the Google Analytics Import do not count towards your hits quota."
```

@Stan-vw @gillnd Data is plural in this context and would make more sense to use `does`, correct?
```suggestion
        "NoteImportNotCountedAsHit": "%1$sNote:%2$s Data imported through the Google Analytics Import does not count towards your hits quota."
```

---

> @Stan-vw @gillnd Data is plural in this context and would make more sense to use `does`, correct?

Actually both are technically correct 'data' plural agrees with 'do' but 'data' singular can be a collective noun and uses 'does'

---

@snake14 This just reads better: Data imported through the Google Analytics Import does not count towards your hits quota

---

Thank you @gillnd My reasoning was off, but it just felt like `does` made more sense. :smile:

---

Added as suggested :+1: